specification/_types/mapping/core.ts

```diff
@@ -99,54 +99,72 @@ export class KeywordProperty extends DocValuesPropertyBase {
   type: 'keyword'
 }
 
-export class NumberPropertyBase extends DocValuesPropertyBase {
+export class IPProperty extends DocValuesPropertyBase {
```

Comment:
Should be `IpProperty`

Reply:
Habits of Go uppercasing acronyms.
I'll change that!